graphql_api/types/mutation/set_upload_token_required/set_upload_token_required.graphql

```diff
@@ -0,0 +1,13 @@
+union SetUploadTokenRequiredError =
+    UnauthenticatedError
+  | UnauthorizedError
+  | ValidationError
+
+type SetUploadTokenRequiredPayload {
+  error: SetUploadTokenRequiredError
+}
+
+input SetUploadTokenRequiredInput {
+  org_username: String!
```

Comment:
Why is this snake and the below camel 👀 could we follow the camel convention pls?

Reply:
Great catch